### PR TITLE
Update dev dependencies

### PR DESCRIPTION
**Pull request by RDIL**
Updates dev dependencies to the latest version.

### CODE REVIEW

**Comment by jessetan** [comment]
Thanks for your PR. We are in the process of moving the build toolchain to webpack, so some of these dependencies will no longer exist or should be updated after the move to webpack.

**Comment by agjohnson** [comment]
Blocking on #771

**Comment by agjohnson** [comment]
#771 is merged, feel free to rebase. I think all of these deps are removed now.

**Comment by RDIL** [comment]
Okay, I'll have dependabot trigger another bump.

File: package.json [hunk 1]
```diff
@@ -10,8 +10,8 @@
   "dependencies": {},
   "devDependencies": {
     "bourbon": "~4.3",
-    "bourbon-neat": "~1.9",
-    "css-loader": "^2.1.1",
+    "bourbon-neat": "~4.0",
```
Comment (agjohnson) [inline package.json]: We can't use > 1.9 current due to Wyrm

**Comment by RDIL** [comment]
Done @agjohnson!

File: package.json [hunk 2]
```diff
@@ -10,8 +10,8 @@
...
+    "css-loader": "^3.1.0",
```
Comment (agjohnson) [inline package.json]: Tested this and things seemed to work :+1:

**Comment by agjohnson** [comment]
Thanks! Looks great! I'll have to note the pin here

**Pull request merged**